posts/2022-02-03-lean-app.md

```diff
@@ -0,0 +1,58 @@
+---
+title: 'The Lean App Movement'
+description: 'Optimize for outcomes that matter – not pointless, repetitive drudgery'
+date: '2022-02-03'
+authors:
+  - Tim Wagner
+---
+
+## Optimize for outcomes that matter – not pointless, repetitive drudgery 
+
+For decades, progress in application design was driven by reducing surface area and shifting complexity to the application’s environment. This progress proved to be cyclical, ebbing and flowing with the pervasive trends of the day: like the shift from mainframes to client/server, which eventually created the server-based Unix era, which itself led to the public cloud, which ultimately gave rise to fully managed and “serverless” offerings. 
+
+The next significant movement in application development has begun. 
+
+The best application developers are leading with judiciousness, maniacally focusing on solving the toughest problems in the leanest, most elegant way possible. As developers, we place a premium on the idea that less is more. We are precision coding. The code we write is entirely aimed at delivering unique value – leaving undifferentiated and previously-solved problems to the rich set of cloud services at our fingertips. 
+
+We are building powerful, lean applications.  
+
+
+### A four-part blog series
+
+This blog series covers the lean app movement in four parts.  
+
+**[Part 1 – Lessons from the Past.](Vendia.net/blog/lean-app-part-1)** This section takes a look at the trendlines that have made it possible to develop lean apps. I highlight the technology shifts that make it possible for us to apply an extreme focus on building software based on customer value, rather than technological ownership
```

Comment:
`Vendia.net/blog/lean-app-part-1` and other links like it need to be full URLs e.g. `https://www.vendia.net/blog/lean-app-part-1`